packages/jmix-react-antd/src/formatters/jmixFront_to_ant.ts

```diff
@@ -71,7 +71,7 @@ export function jmixFront_to_ant(
 
       if (typeof value === 'object') {
         // Recursively traverse child entity
-        fields[attrName] = jmixFront_to_ant(value, propInfo.type, metadata);
+        fields[attrName] = getCompositionAttrValue(value, propInfo.type, metadata);
```

Comment:
maybe call `jmixFront_to_ant()` without helpers, to make recursion calling more explicitly
```suggestion
        fields[attrName] = {
          ...jmixFront_to_ant(entityInstance, entityName, metadata),
          _instanceName: entityInstance._instanceName
        }
```

Reply:
or rename `getCompositionAttrValue()` in such a way that the name looks like `jmixFront_to_ant()`
```suggestion
        fields[attrName] = jmixFront_to_ant_composition(value, propInfo.type, metadata);
```

Reply:
@HegelPro i used helper function, cause this code block is using several time, according DRY. I am not sure, that another naming can do recursion calling more explicitly.
@vyacheslav-pushkin what do you think?